net-libs/xdp-tools/xdp-tools-1.4.1.ebuild

```diff
@@ -33,22 +33,36 @@ QA_PREBUILT="usr/lib/bpf/*.o"
 MAKEOPTS+=" V=1"
 
 PATCHES=(
-	"${FILESDIR}"/1.4.1-no-Werror.patch
+	"${FILESDIR}"/1.4.1-fix-memory-leak-in-xsk_setup_xdp_prog.patch
 )
 
+src_prepare() {
+	# remove -Werror: #899744
+	sed -i 's/-Werror//g' lib/Makefile lib/defines.mk
```

Comment:
Missing `|| die` on external commands (all the seds here).

Reply:
:heavy_check_mark: